pom.xml

```diff
@@ -10,7 +10,7 @@
 
     <groupId>com.rebuy.archetypes</groupId>
     <artifactId>rebuy-silo-archetype</artifactId>
-    <version>9.9.1-SNAPSHOT</version>
+    <version>10.0.0-SNAPSHOT</version>
```

Comment:
that should be done by the maven release plugin

Reply:
True.